### PR TITLE
Added Trasportation Runbook, and section in the Platform Runbook

### PR DESCRIPTION
**Pull request by davidajuan**

### CODE REVIEW

File: docs/TESTING-RUNBOOK.md [hunk 1]
```diff
@@ -7,6 +7,15 @@ permalink: /testing-runbook
 
 The Testing Runbook walks you through the Drive Through Testing site logisitcs, broken down by the two main checkpoints.  The roles required for each checkpoint are provided, and a more comprehensive staffing plan can be found [here](./site-staffing-plan)
 
+## Employee Entrance Screening Tent
+
+This station is staffed by 2 nurses or nurse assistant.  Their role is to take every person's temperature coming in through the employee entrance (full time staff, volunteers, medical staff, deliveries, media, maintenance and police / EMS)
```
Comment (onelight) [inline docs/TESTING-RUNBOOK.md:12]: nurse assistants.

Comment (onelight) [reply docs/TESTING-RUNBOOK.md:12]: take the temperature of each person coming in...

Comment (onelight) [reply docs/TESTING-RUNBOOK.md:12]: missing a period at the end of line 12.

File: docs/TESTING-RUNBOOK.md [hunk 2]
```diff
@@ -7,6 +7,15 @@ permalink: /testing-runbook
 
 The Testing Runbook walks you through the Drive Through Testing site logisitcs, broken down by the two main checkpoints.  The roles required for each checkpoint are provided, and a more comprehensive staffing plan can be found [here](./site-staffing-plan)
 
+## Employee Entrance Screening Tent
+
+This station is staffed by 2 nurses or nurse assistant.  Their role is to take every person's temperature coming in through the employee entrance (full time staff, volunteers, medical staff, deliveries, media, maintenance and police / EMS) 
+
+- The nurse/nurse assistant takes each person's temperature as they come through the checkpoint. Your temperature must be under 100.1 to be allowed entrance.  A color coded sticker with the current date is provided to each person that passes the temperature check.
```
Comment (onelight) [inline docs/TESTING-RUNBOOK.md:14]: color-coded

File: docs/TESTING-RUNBOOK.md [hunk 3]
```diff
@@ -7,6 +7,15 @@ permalink: /testing-runbook
...
+- If someone is at 100.1 or over, the nurses are required to call the project manager and report it.  The project manager then calls the employee's supervisor and notifies them that their employee has a temperature.
```
Comment (onelight) [inline docs/TESTING-RUNBOOK.md:15]: Nurses are required to report all fevers over 100.1 to the project manager.

File: docs/TESTING-RUNBOOK.md [hunk 4]
```diff
@@ -7,6 +7,15 @@ permalink: /testing-runbook
...
 The Testing Runbook walks you through the Drive Through Testing site logisitcs, broken down by the two main checkpoints.  The roles required for each checkpoint are provided, and a more comprehensive staffing plan can be found [here](./site-staffing-plan)
 
+## Employee Entrance Screening Tent
+
+This station is staffed by 2 nurses or nurse assistant.  Their role is to take every person's temperature coming in through the employee entrance (full time staff, volunteers, medical staff, deliveries, media, maintenance and police / EMS) 
+
+- The nurse/nurse assistant takes each person's temperature as they come through the checkpoint. Your temperature must be under 100.1 to be allowed entrance.  A color coded sticker with the current date is provided to each person that passes the temperature check.
+- If someone is at 100.1 or over, the nurses are required to call the project manager and report it.  The project manager then calls the employee's supervisor and notifies them that their employee has a temperature. 
+- The staff for this tent arrive two hours before the testing site opens to the public.  
+- PPE: Mask, gloves, goggles, and touch free thermometers are required.
```
Comment (onelight) [inline docs/TESTING-RUNBOOK.md:17]: PPE (mask, gloves, goggles) and touch-free thermometers are required.

File: docs/TRANSPORTATION.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+title: Transportation
+description: Overview of the Transportation Program
+permalink: /transportation
+---
+
+In order to ensure all Detroit residents have access to the COVID-19 testing site, a transportation service was stood up to pick-up residents at their home, take them through the testing site, drop residents back off at their residence, and then sanitize the vehicle.
+
+How it works:
+
+1. Residents that have met the qualification requirements contact the COVID-19 Test Site call center to request an appointment. If a resident requests transportation during this call, the call center representative connects the resident to the transportation call center for the resident to book a ride directly with transportation vendor.
```
Comment (onelight) [inline docs/TRANSPORTATION.md:12]: the call center representative connects the resident to the transportation call center to book a ride directly with transportation vendor.

File: docs/TRANSPORTATION.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+3.	The transportation provider picks up residents at their home and drives them to and from the testing site.
+
+4.	Residents pay $2 in cash for the ride by placing it in a sealed plastic bag and leaving it in the back seat of the car. However, if a resident does not have $2, they will still get a ride, and the City will reimburse the taxi provider. 
+
+5.	Vehicles are cleaned in between each use at one of two cleaning sites the City set up with an outside cleaning vendor. 
+
+It is imperative that safety for both the driver and the passenger is prioritized in this process.  This has been accomplished through the following method:
+
+- The City provides PPE for both the driver and passenger. The driver is provided with an N-95 masks, goggles, and gloves.   The passenger is provided with a supply surgical masks and gloves.  Passengers are informed they will be required to wear a mask and gloves when they book the ride and that they will be refused a ride if they do not wear the PPE.
```
Comment (onelight) [inline docs/TRANSPORTATION.md:24]: The City provides PPE for both the driver and passenger. The driver is provided with an N-95 mask, goggles, and gloves. The passenger is provided with a supply surgical mask and gloves.  Passengers are informed that they will be required to wear a mask and gloves when they book the ride and that they will be refused a ride if they do not wear the PPE.

File: docs/TRANSPORTATION.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+- Safety and sanitation protocols were developed that drivers and riders must follow. These protocols are posted in the back seat for riders to view while in transport. 
+- A daily checklist was provided for drivers to use to ensure they have all of the PPE and sanitation materials needed.
+- Only vehicles with a physical separation between the driver and the passenger are used in the service..
```
Comment (onelight) [inline docs/TRANSPORTATION.md:27]: remove double period

File: docs/TRANSPORTATION.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+Intelliride is the vendor that the City worked with to oversee the program, dispatch and coordinate rides with multiple vendors, and provide cleaning protocols to ensure riders and drivers are safe and protected. On April 6, 2020, we started providing rides to any Detroit resident who had a prescription to get a test. On April 18, 2020, we expanded this program to provide essential employees COVID-19 tests, and a ride to the testing site if requested.
```
Comment (onelight) [inline docs/TRANSPORTATION.md:29]: On April 18, 2020, we expanded this program to include essential employees.

**Comment by davidajuan** [comment]
@onelight take a look now.

**Pull request merged**